make/tools.mk

```diff
@@ -109,6 +109,10 @@ ifdef WINDOWS
 	$(V1) ./downloads/qt-opensource-windows-x86-mingw492-5.5.0.exe
 endif
 
+	# Configure Qt Creator user project files (GCS and flight firmware)
```

Comment:
Can this be a separate make target. I'd rather not conflate installing the Qt SDK with setting up an environment. It's also nice if (e.g. right now me) would like to say
```make qtcreator_environment```